### DISCUSSION TITLE
"Unbypassable" disk space quota

### DISCUSSION THREAD

**Discussion by caukub** [pelican-dev/panel#23] [Ideas]
Currently the assigned disk space limit for service (container) can be easily exceeded. This can be solved by using filesystem quotas or virtual disks. ([Docker supports limiting disk space for some storage drivers](https://docs.docker.com/reference/cli/docker/container/run/#storage-opt))

**Comment by The0mikkel** [comment]
Just another Pterodactyl user here. 
Are you thinking of game server disk usage?

I have had game servers shut down, when they reached the max disk usage specified in Pterodactyl, as well as not being able to start them again.

**Comment by parkervcp** [comment]
The docker options for limiting disk space do not work on bind mounts which is how we mount the container home folder.

**Comment by iamkubi** [comment]
Already an issue for this https://github.com/pelican-dev/panel/issues/119

**Comment by lancepioch** [comment]
Thanks, please follow #119
